TrafficCrash/Models/TrafficCrashPeople.cs

```diff
@@ -17,48 +17,65 @@ namespace TrafficCrashPeoples
 
     public partial class TrafficCrashPeople
     {
+        private string person_id;
+        private string person_type;
+        private string rd_no;
+        private long vehicle_id;
+        private DateTimeOffset crash_date;
+        private string sex;
+        private string safety_equipment;
+        private string airbag_deployed;
+        private string ejection;
+        private string injury_classification;
+        private string driver_action;
+        private string driver_vision;
+        private string physical_condition;
+        private string bac_result;
+
+
+
         [JsonProperty("person_id")]
-        public string PersonId { get; set; }
+        public string PersonId { get => person_id; set => person_id = value; }
```

Comment:
While encapsulation is always a good idea, the old way is fine for our purposes.  .net considers { get; set; } to be a shortcut for encapsulation.